CMakeLists.txt

```diff
@@ -15,6 +15,9 @@ option(BUILD_OPCUA "Set to ON to compile with OPC-UA protocol support" OFF)
 option(BUILD_SYSTEMD "Set to ON to compile with systemd additional support" ON)
 option(INSTALL_LIB "Set to ON to install library and headers" OFF)
 
+set(logger_type "boost" CACHE STRING "")
```

Comment:
nitpick: should we use upper case names, like all the options that we have?

Reply:
Yes, probably. Aesthetically I like lower case for individual module configuration better, but there is no real reason to introduce inconsistency.